client/apps/edit2/components/content/sections/footer/test/index.test.js

```diff
@@ -11,9 +11,9 @@ describe('SectionFooter', () => {
 
   beforeEach(() => {
     props = {
-      article: new Backbone.Model(FeatureArticle),
+      article: clone(FeatureArticle),
```

Comment:
This probably doesn't affect the test but it might be preferred to use cloneDeep over clone for objects with deep nesting as clone only does a shallow copy.

Reply:
Sounds good, will update that now